shared/tracker2/assertion/index.js

```diff
@@ -396,7 +396,12 @@ class Assertion extends React.PureComponent<Props, State> {
 }
 
 const styles = Styles.styleSheetCreate({
-  container: {flexShrink: 0, paddingBottom: 4, paddingTop: 4},
+  container: Styles.platformStyles({
+    common: {flexShrink: 0, paddingBottom: 4, paddingTop: 4},
+    isElectron: {
+      ...Styles.desktopStyles.windowDraggingClickable,
```

Comment:
:sheep: might be a better separation of concerns if we put this in `styles.assertions` in `tracker2/index.desktop` if that also works.